src/snowflake/snowpark/_internal/telemetry.py

```diff
@@ -359,3 +363,15 @@ def send_copy_pattern_telemetry(self):
         self.send_function_usage_telemetry(
             "copy_pattern", TelemetryField.FUNC_CAT_COPY.value
         )
+
+    def send_sql_simplifier_telemetry(self, session_id: str, sql_simplifier_enabled: bool) -> None:
+        message = {
+            **self._create_basic_telemetry_data(
+                TelemetryField.TYPE_FUNCTION_USAGE.value
```

Comment:
I think we shall introduce a new type of telemetry: TYPE_SIMPLIFIER_UAGE, the function usage is for telemetry of api calls.